web/frontend/src/pages/election/components/Question.tsx

```diff
@@ -1,168 +1,69 @@
-                onChange={handleChange}
-                name="MaxN"
-                min={MinN}
-                type="number"
-                placeholder="Enter the MaxN"
-                className="my-1 w-60 ml-1 border rounded-md"
-              />
-              <label className="block text-md font-medium text-gray-500">MinN</label>
-              <input
-                value={MinN}
-                onChange={handleChange}
-                name="MinN"
-                max={MaxN < MAX_MINN ? MaxN : MAX_MINN}
-                min="0"
-                type="number"
-                placeholder="Enter the MinN"
-                className="my-1 w-60 ml-1 border rounded-md"
-              />
-              {showExtraFields(question)}
-              <label className="block text-md font-medium text-gray-500">Choices</label>
-              {Choices.map((choice: string, idx: number) => (
-                <div key={`${ID}wrapper${idx}`}>
-                  <input
-                    key={`${ID}choice${idx}`}
-                    value={choice}
-                    onChange={updateChoice(idx)}
-                    name="Choice"
-                    type="text"
-                    placeholder="Enter your choice"
-                    className="my-1 w-60 ml-1 border rounded-md"
-                  />
-                  <button
-                    key={`${ID}deleteChoice${idx}`}
-                    type="button"
-                    className="inline-flex ml-2 items-center border border-transparent rounded-md shadow-sm text-sm font-medium text-white bg-indigo-600 hover:bg-indigo-700"
-                    onClick={deleteChoice(idx)}>
-                    <MinusSmIcon className="h-4 w-4" aria-hidden="true" />
-                  </button>
-                </div>
-              ))}
-              <button
-                type="button"
-                className="flex p-2 h-8 w-8 mb-2 rounded-md bg-green-600 hover:bg-green-800 sm:-mr-2"
-                onClick={addChoice}>
-                <PlusSmIcon className="h-5 w-5 text-white" aria-hidden="true" />
-              </button>
+    <div className="pl-3">
+      <AddQuestionModal
+        open={openModal}
+        setOpen={setOpenModal}
+        notifyParent={notifyParent}
+        question={question}
+        handleClose={() => setOpenModal(false)}
+      />
+      <div className="flex flex-row justify-between w-full h-24 ">
+        <div className="flex flex-col max-w-full pl-2">
+          <div className="mt-3 flex">
+            <div className="h-9 w-9 rounded-full bg-gray-100 mr-2 ml-1">
+              <DisplayTypeIcon Type={Type} />
             </div>
-          </Disclosure.Panel>
+            <div className="pt-1.5 max-w-md pr-8 truncate">
+              {Title.length ? Title : `Enter ${Type} title`}
+            </div>
+          </div>
+
+          <div className="flex mt-2 ml-2">
+            <div className="ml-8">{Type.charAt(0).toUpperCase() + Type.slice(1)}</div>
```

Comment:
I still think this is not necessary to display the question type in plain text, as there is already the icon.

Reply:
I think we made the redundancy on purpose in case someone doesn't understand the icon @cmsigrist .